Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/pageObjects/AddressesListPage.java

```diff
@@ -0,0 +1,46 @@
+package com.griddynamics.qa.vikta.uitesting.sample.pageObjects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+public class AddressesListPage extends BasePage{
+
+    @FindBy(css = "#tblAddresses > tbody > tr:first-child > td:nth-child(2) > a")
+    public WebElement firstAddressHyperlink;
```

Comment:
if all these WebElements are accessed only in this class, please make them private